### ISSUE TITLE
Gateways and Openers constraints don't work through Config UI X

### ISSUE THREAD

**Issue by SafariKC** [hjdhjd/homebridge-myq2#25]
The current descriptors for Config UI are forcing the Arrays to come in quoted.  Which isn't ready by the plugin appropriately.  I'm hoping you an change the descriptor in Config UI X to be able to have the output in the config look like this instead which is much cleaner: 

 ```
       "gateways": [
                6702872
            ],
            "openers": [
                9910182,
                9910311
            ],
```

Today the plugin puts it in this format which doesn't actually constrain anything as it is not an array, it's a string. 

 ```
       "gateways": "[6702872]",
       "openers": "[9910182,9910311]",

```

**Comment by hjdhjd** [comment]
Fixed. Update away.

**Comment by SafariKC** [comment]
I’m pretty sure that Config UI X is still not doing the right thing.  But I’ve manually updated my Config to get around it it’s not a big deal but others could find it a stumbling block.

**Comment by hjdhjd** [comment]
@SafariKC Can you elaborate on what's going on? Is the first issue still what's occurring? Let me know and I'll take a look.

**Comment by SafariKC** [comment]
Quotes are still showing up around the array.  Enter some data following the sample data and hit save you’ll see

**Comment by hjdhjd** [comment]
Update coming shortly.

**Comment by hjdhjd** [comment]
New update out...I refactored the UI. Should address this moving forward. Thanks for bringing it to my attention.

**Comment by SafariKC** [comment]
Looks good now! Much better!

**Comment by SafariKC** [comment]
Actually it's not... gateways is still wrapping the array in ""

**Comment by hjdhjd** [comment]
Please share screenshots and your config.json. Before doing so, you may want to try a clean config and ensure you're on the latest, etc.

**Comment by SafariKC** [comment]
I have tried a clean config each time.  And I've uninstalled and re-installed the Plugin. 

The Array Objects are being saved as Strings and not Numbers which your plugin requires. 

Here's screenshots of the bad behavior and the entry of info in the plugin... the fix is to remove the ""'s. 
<img width="775" alt="Screen Shot 2020-06-28 at 5 45 35 PM" src="https://user-images.githubusercontent.com/20827467/85959022-a9d06b80-b967-11ea-954d-511081a9b626.png">
<img width="399" alt="Screen Shot 2020-06-28 at 5 46 13 PM" src="https://user-images.githubusercontent.com/20827467/85959024-accb5c00-b967-11ea-9142-339c06078cc6.png">
